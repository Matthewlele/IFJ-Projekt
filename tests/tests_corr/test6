varFloat = 5.6
varInt = 4

varInt = varFloat + 5 / varInt * 5 / "ahoj"

print varInt + varFloat
print "\n"
print varInt
