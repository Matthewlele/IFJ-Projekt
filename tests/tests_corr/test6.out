expected SEMANTIC ERR: line 4
